### PR TITLE
Added a log

### PR DESCRIPTION
**Pull request by RichardLitt**
As noted by @diasdavid in https://github.com/ipfs/webui/pull/130#discussion_r48386312

### CODE REVIEW

File: app/scripts/utils/localStorage.js [hunk 1]
```diff
@@ -30,7 +30,9 @@ function doOperation (op) {
     if (op === 'getItem') {
       try {
         result = JSON.parse(result)
-      } catch (e) {}
+      } catch (e) {
+        console.log('LocalStorage error: Unable to parse result as json. Message: ' + e.message)
```
Comment (Dignifiedquire) [inline app/scripts/utils/localStorage.js]: Please use `debug` for logging

**Comment by RichardLitt** [comment]
How does this look?

File: app/scripts/utils/localStorage.js [hunk 2]
```diff
@@ -1,4 +1,7 @@
 import _ from 'lodash'
+import Debug from 'debug'
+
+const debug = Debug('ipfs:utils:LocalStorage')
```
Comment (Dignifiedquire) [inline app/scripts/utils/localStorage.js]: Can we use the convention I have in #155, i.e.

```js
import debug from 'debug'

const log = debug('utils:localStorage')
```

**Comment by RichardLitt** [comment]
Sure. I got that from /Users/richard/src/ipfs-webui/app/scripts/views/globe.js, which doesn't seem to have the same convention. Maybe we should make a styleguide? Also, there's a console.warn on the same page...

**Comment by Dignifiedquire** [comment]
@RichardLitt yeah that all should be fixed in #155

File: app/scripts/utils/localStorage.js [hunk 3]
```diff
@@ -30,11 +33,13 @@ function doOperation (op) {
...
-      } catch (e) {}
+      } catch (e) {
+        log('Unable to parse result as json. Message: %s', e.message)
+      }
     }
 
     return result
   } catch (e) {
-    console.warn('LocalStorage error: ' + e.message)
+    console.warn(e.message)
```
Comment (Dignifiedquire) [inline app/scripts/utils/localStorage.js]: Let's use `debug` here as well

Comment (Dignifiedquire) [inline app/scripts/utils/localStorage.js]: But with a warn level: `const warn = debug('utils:localStorage:warn')` or sth like that

Comment (RichardLitt) [inline app/scripts/utils/localStorage.js]: Done.

**Comment by Dignifiedquire** [comment]
LGTM :)

**Pull request merged**